Package.swift

```diff
@@ -566,3 +572,15 @@ for index in package.targets.indices {
     package.targets[index].swiftSettings = .pointFreeSettings
   }
 }
+
+func transcripts() -> [Resource] {
+  let transcriptsDirectoryPath = URL(fileURLWithPath: #filePath)
+    .deletingLastPathComponent()
+    .appendingPathComponent("Sources")
+    .appendingPathComponent("Transcripts")
+    .appendingPathComponent("Resources")
+    .path
+
+  return try! FileManager.default.contentsOfDirectory(atPath: transcriptsDirectoryPath)
+    .map { .copy("Resources/\($0)") }
+}
```

Comment:
I was able to copy over just the markdown files that are in the `Resources` directory.